kraken/kubernetes/client.py

```diff
@@ -22,6 +28,19 @@ def list_nodes():
     return nodes
 
 
+# List nodes in the cluster that can be killed
+def list_killable_nodes():
```

Comment:
It would be nice if this function could be modified to return killable `Ready` nodes. After the node chaos injection, the node might not come back to Ready state. In that case the node which is in NotReady state is not considered for next chaos scenario injection.

Reply:
Added, please take a look